### DISCUSSION TITLE
How to fill

### DISCUSSION THREAD

**Discussion by ihassona** [ebarellar/LFColoringBook#10] [General]
Hello
Thanks for this helpful source.  But I want to ask if I can fill some area by one click not by moving around?
Regards

**Comment by ebarellar** [comment]
Not at the moment but it's a fun little project to implement, and it's not that hard

If you want to implement it by "tapping" then you need to retrieve the mask at touches began exactly the same way as right now. Then you need to find a way to differentiate between a "tap" and a stroke gesture (that it's all up to you)
Once you have the event that indicates the end of a successful tap then just clip the frozentContext with the current mask and fill a rect based on the bounding box of the mask, it should "fill" the proper area

Something like that would end up breaking the strokes history routine as it exists right now, it would need to be modified to differentiate between strokes and taps
